packages/autorest.typescript/test/rlcIntegration/lroRest.spec.ts

```diff
@@ -1001,7 +1001,7 @@ describe("LRO Rest Client", () => {
       assert.equal(result.status, "200");
     });
 
-    it("should handle put200InvalidJson", async () => {
+    it.only("should handle put200InvalidJson", async () => {
```

Comment:
![image](https://github.com/Azure/autorest.typescript/assets/98569699/3bc6d9fc-d262-4ad2-bca9-edca0d271172)
repro ci test failure in my local env.

step:
1. start a **git bash**
2. run ```rush update && rush build```
3. ```cd package/autorest.typescript```
4. run ```rushx start-test-server:v2```
5. start a new **git bash** again
6. ```cd package/autorest.typescript```
7. run ```rushx rlc-integration-test:alone```
8. run ```rushx stop-test-server```